src/OpenTelemetry.Contrib.Extensions.AWSXRay/Resources/ResourceDetectorUtils.cs

```diff
@@ -37,11 +39,13 @@ internal static async Task<string> SendOutRequest(string url, string method, Key
             httpRequestMessage.Method = new HttpMethod(method);
             httpRequestMessage.Headers.Add(header.Key, header.Value);
 
+#pragma warning disable CA2000 // Dispose objects before losing scope
             var httpClient = handler == null ? new HttpClient() : new HttpClient(handler);
```

Comment:
Wouldn't this be better here?

```suggestion
           using var httpClient = handler == null ? new HttpClient() : new HttpClient(handler);
```